### PR TITLE
Added BeSman ML assessment environment

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/besecure-ce-env-repo/issues/40

Added CounterFit and Watchtower installation and uninstallation script

### CODE REVIEW

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# If you wish to update the default configuration values, copy this file and place it under your home dir, under the same name.
+# These variables are used to drive the installation of the environment script.
+# The variables that start with BESMAN_ are converted to environment vars.
+# If you wish to add any other vars that should be used globally, add the var using the below format.
+# BESMAN_<var name>: <value>
+# If you are not using any particular value, remove it or comment it(#).
+#*** - These variables should not be removed, nor left empty.
+
+# BESMAN_ARTIFACT_TYPE - project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: # project/ml model/training dataset #***
```
Comment (asa1997) [inline ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml:11]: Should be ml model

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# BESMAN_ARTIFACT_NAME - name of the artifact under assessment.
+BESMAN_ARTIFACT_NAME: ml_assessment-env #***
```
Comment (asa1997) [inline ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml:14]: Should be the name of the artifact under test. Should be empty for now.

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# BESMAN_ARTIFACT_VERSION - version of the artifact under assessment.
+BESMAN_ARTIFACT_VERSION: #Enter the version of the artifact here. #***
+
+# BESMAN_ARTIFACT_URL - Source code url of the artifact under assessment.
+BESMAN_ARTIFACT_URL: https://github.com/Be-Secure/ml_assessment-env #***
+
+#BESMAN_ENV_NAME - This variable stores the name of the environment file.
+BESMAN_ENV_NAME: ml_assessment-env #***
```
Comment (asa1997) [inline ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml:23]: Should this be ml_assessment-env or ml_RT-env? Please confirm.

**Review by asa1997 — CHANGES_REQUESTED** [review]

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml (hunk 1 above)
Comment (pramit-d) [reply ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml]: Done

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml (hunk 2 above)
Comment (pramit-d) [reply ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml]: Updated.

File: ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml (hunk 3 above)
Comment (pramit-d) [reply ml_assessment/0.0.1/besman-ml_assessment-env-config.yaml]: Updated the BeSman env name to 'ML-RT-env'

**Review by asa1997 — APPROVED** [review]
@pramit-d LGTM

**Pull request merged**